### PR TITLE
adds an example to load schema and validate in README

### PR DESCRIPTION
**Pull request by desaikd**
*Issue #80*

*Description of changes:*
This PR adds an example to load schema and validate in README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,91 @@ An implementation of [Amazon Ion Schema](http://amzn.github.io/ion-schema) in Ru
 
 **This package is considered experimental, under active/early development, and the API is subject to change.**
 
+## Getting Started
+
+The following rust code sample is a simple example of how to use this API. The `my_int_type` is an integer ion type. 
+A sample schema `my_schema.isl` can be loaded using `ion-schema-rust` as following:
+
+### Example schema `my_schema.isl`
```
Comment (zslayton) [inline README.md:16]: Consider moving some of the exposition about the schema itself into the schema subsection.

```suggestion
## Getting Started

The following rust code sample is a simple example of how to use this API.

### Example schema `my_schema.isl`

This file (`my_schema.isl`) defines a new type (`my_int_type`) based on Ion's `int` type.
```
The section starting on line 32 (where the schema is loaded/used) already says that's what it's doing, so I removed this bit:
> A sample schema `my_schema.isl` can be loaded using `ion-schema-rust` as following:

File: README.md [hunk 2]
```diff
@@ -8,6 +8,91 @@ An implementation of [Amazon Ion Schema](http://amzn.github.io/ion-schema) in Ru
...
+type::{
+  name: my_int_type,
+  type: int,
+}
+
+schema_footer::{
+}
+```
+
+### Simple rust example to demonstrate how to load a schema and validate an ion value
```
Comment (zslayton) [inline README.md:31]:
```suggestion
### Loading a schema and validating an Ion value
```

File: README.md [hunk 3]
```diff
@@ -8,6 +8,91 @@ An implementation of [Amazon Ion Schema](http://amzn.github.io/ion-schema) in Ru
...
+    )];
+
+    // Create a new schema system from given document authorities
+    let mut schema_system = SchemaSystem::new(document_authorities);
+
+    // Provide schema id for the schema you want to load (schema_id is the schema file name here)
+    let schema_id = "my_schema.isl";
+
+    // Load schema
+    let schema: Rc<Schema> = schema_system.load_schema(schema_id)?;
```
Comment (zslayton) [inline README.md:55]: As a TODO, we really should have a public type that hides the `Rc` implementation detail.

**Review by zslayton — APPROVED** [review]

**Review by popematt — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (desaikd) [reply README.md:55]: Yes, there is already an issue opened to track this: https://github.com/amzn/ion-schema-rust/issues/36

**Pull request merged**